app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+  def create
+    user = User.find_by email: params[:session][:email].downcase
```

Comment:
```suggestion
    user = User.find_by email: params.dig(:session, :email)&.downcase
```
em tìm hiểu method `dig` nhé bé

Reply:
vâng chị

Reply:
áp dụng luôn đi. dùng dig thì khác gì